src/shared/storage/redux-pesist.ts

```diff
@@ -114,7 +114,7 @@ export const persistConfig: PersistConfig<RootState> & UntypedDeserializeOption
     'analytics',
     'chains',
     'ordinals',
-    'keys',
+    'software-keys',
```

Comment:
This rename might add more complexity than worth, I think

Reply:
yeah, there was some refactoring complexity, but still I believe just naming this `keys` is rather misleading